clouddriver-openstack/src/main/groovy/com/netflix/spinnaker/clouddriver/openstack/deploy/ops/servergroup/DeployOpenstackAtomicOperation.groovy

```diff
@@ -177,4 +184,20 @@ class DeployOpenstackAtomicOperation implements AtomicOperation<DeploymentResult
     result
   }
 
+  /**
+   * Return the file contents of a template, either from the account config location or from the classpath.
+   * @param filename
+   * @return
+   */
+  String getTemplateFile(String filename) {
+    String template = ""
+    String tmplDir = description.credentials.credentials.heatTemplateLocation
```

Comment:
This will read the path off the config option and then slurp in the file that already exists on disk, correct? Just want to ensure that the specified path isn't being passed in through the request.